app/serializers/discourse_post_event/invitee_serializer.rb

```diff
@@ -15,5 +15,14 @@ def include_id?
     def user
       BasicUserSerializer.new(object.user, embed: :objects, root: false)
     end
+
+    def meta
```

Comment:
Im returning these info in meta so I can properly update the model without any other remote calls when changing attendance